src/main/java/com/toptrumps/core/engine/Game.java

```diff
@@ -112,4 +120,8 @@ private int getRandomInteger(int min, int max) {
         return randomGenerator.nextInt((max - min) + 1) + min;
     }
 
+    public boolean getHumanRemoved() {
+        return humanRemoved;
+    }
+
```

Comment:
The logic is sound but the naming is a bit unclear - maybe something something like getHumanStatus? Also maybe something like humanIsRemoved for the variable? Just to help make everything really obvious